### DISCUSSION TITLE
Directus landing page stacks & template

### DISCUSSION THREAD

**Discussion by liho00** [directus/directus#10150] [Archive]
Directus landing looks awesome, may i know whats the stacks behind it, does this landing page also using directus as CMS? Besides, izit it using vue js or next js as frontend framework, etc. Izit possible to open source the template?

Thanks in advance.

**Comment by rijkvanzanten** [comment]
> Directus landing looks awesome

Thanks! ❤️ 

> may i know whats the stacks behind it

Sure thing. It's a statically generated website using [Eleventy](https://www.11ty.dev).

> does this landing page also using directus as CMS

Absolutely! All 'dynamic' sections (like articles and the forms) are pulling from/to a Directus instance.

> Besides, izit it using vue js or next js as frontend framework, etc.

No. There's not really any interactivity on the website, so there's no need for a full framework.

> Izit possible to open source the template?

Ehh the codebase itself isn't really ready to be open sourced. Besides, there aren't a lot of changes to be made by the community (unlike the docs), as it's fairly static.

That being said, we are working on an examples repo that contains pre-made examples of how to integrate Directus with various build tools, including Eleventy 🙂

**Reply by liho00** [reply]
Thanks for your explanation! 

> That being said, we are working on an examples repo that contains pre-made examples of how to integrate Directus with various build tools, including Eleventy 🙂

That's great looking forward to it!

**Reply by botzill** [reply]
Hi @rijkvanzanten, do you have any updates on the "we are working on an examples repo that contains pre-made examples of how to integrate Directus with various build tools, including Eleventy" ?

**Reply by rijkvanzanten** [reply]
> Hi @rijkvanzanten, do you have any updates on the "we are working on an examples repo that contains pre-made examples of how to integrate Directus with various build tools, including Eleventy" ?

https://github.com/directus/examples 🙂

**Reply by botzill** [reply]
I tried `Eleventy` and in env there is DIRECTUS_STATIC_TOKEN, what do I set this as? I set to user admin token, but I get

`You don't have permission to access this. (via Error)`

**Reply by rijkvanzanten** [reply]
Hmm that's a little hard to tell from the information provided. Mind opening an issue on the examples repo detailing exactly what you've tried so far? Thanks! 🙂
